client/index.html

```diff
@@ -9,6 +9,7 @@
 <body>
   <script>
     deeplink.createBrowser('com.android.chrome');
+    register();
```

Comment:
I think this is not correct.
This function should be called in client/wedding/index.html
/cc @nadongguri